.github/workflows/pr_bench.yml

```diff
@@ -0,0 +1,14 @@
+name: Benchmark pull requests
+on: [pull_request]
+jobs:
+  runBenchmark:
+    name: run benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: boa-dev/criterion-compare-action@v3
+        with:
+          # Optional. Compare only this benchmark target
+          benchName: "end2end_benchmark"
```

Comment:
This must be one of the targets in the `Cargo.toml`. I suspect you want end2end.